Project 1/styles.css

```diff
@@ -0,0 +1,20 @@
+.card{
+        max-width: 800px;
+        margin: 30px auto;
+        background-color: #ffffff;}
+
+img{
+    max-width: 100%;}
+    title {
```

Comment:
The `title` element in HTML is in the `head` and therefore does not appear on the page -- controls the page title in the browser tabs, history and favorites, but is not the same thing as a heading (e.g., `h1`, `h2`) inside the `body` and visible in the browser. As a result, you should either delete lines 8-11 (because `title` cannot be styled) or change it to be your headline (e.g., `h1`).